packages/css/alert.css

```diff
@@ -1,69 +1,73 @@
 .ds-alert {
+  --dsc-alert-background: var(--ds-color-info-surface-default);
   --dsc-alert-border-color: var(--ds-color-info-border-strong);
   --dsc-alert-border-radius: min(1rem, var(--ds-border-radius-md));
   --dsc-alert-color: var(--ds-color-neutral-text-default);
-  --dsc-alert-icon-color: var(--ds-color-neutral-text-subtle);
+  --dsc-alert-font-size: var(--ds-font-size-5);
+  --dsc-alert-gap: var(--ds-spacing-2);
+  --dsc-alert-icon-color: var(--ds-color-info-text-subtle);
   --dsc-alert-icon-size: var(--ds-sizing-7);
-  --dsc-alert-background: var(--ds-color-info-surface-default);
+  --dsc-alert-icon-url: url("data:image/svg+xml,%3Csvg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' d='M3.25 4A.75.75 0 0 1 4 3.25h16a.75.75 0 0 1 .75.75v16a.75.75 0 0 1-.75.75H4a.75.75 0 0 1-.75-.75zM11 7.75a1 1 0 1 1 2 0 1 1 0 0 1-2 0m-1.25 3a.75.75 0 0 1 .75-.75H12a.75.75 0 0 1 .75.75v4.75h.75a.75.75 0 0 1 0 1.5h-3a.75.75 0 0 1 0-1.5h.75v-4h-.75a.75.75 0 0 1-.75-.75'/%3E%3C/svg%3E");
   --dsc-alert-padding: var(--ds-spacing-6);
 
-  border: 1px solid var(--dsc-alert-border-color);
-  border-radius: var(--dsc-alert-border-radius);
   background: var(--dsc-alert-background);
+  border-radius: var(--dsc-alert-border-radius);
+  border: 1px solid var(--dsc-alert-border-color);
   color: var(--dsc-alert-color);
-  padding: var(--dsc-alert-padding);
-  display: grid;
-  grid-auto-flow: column;
-  grid-auto-columns: min-content auto;
-  gap: var(--ds-spacing-2);
-}
-
-.ds-alert__icon {
-  height: var(--dsc-alert-icon-size);
-  width: var(--dsc-alert-icon-size);
-  color: var(--dsc-alert-icon-color);
-}
-
-.ds-alert__content {
-  display: flex;
-  flex-direction: column;
-}
+  font-size: var(--dsc-alert-font-size);
+  line-height: var(--ds-line-height-md);
```

Comment:
I suggest we **keep** `asChild` on typography elements until we can compose. We are missing a couple things for font here, which would not be missing if we use `asChild`.

Reply:
Will fix - dette var nok en misforståelse fra sist møte :)